Makefile

```diff
@@ -56,20 +60,35 @@ tags:
 
 fetch_build:
 	curl -Lk -o $(ARCHIVE_PATH) $(BUILD_URL)/$(ARCHIVE_FILENAME)
+	if [ ! -z "$(DEPS_URL)" ]; then                                          \
+		set -e;                                                              \
+		FILENAME=$$(curl -k --list-only $(DEPS_URL)/repo/ | grep chroma-manager-integration | sed -e 's/.*\(chroma-manager-integration-[^/]*\)\/.*/\1/'); \
+		echo $$FILENAME;                                                     \
+		curl -Lk -o chroma-manager/$$FILENAME $(DEPS_URL)/repo/$$FILENAME;   \
+	fi
 
 # build the chroma-{agent,management} subdirs before the chroma-dependencies subdir
 chroma-dependencies: chroma-agent chroma-manager chroma-diagnostics
 chroma-bundles: chroma-dependencies
 
 destroy_cluster: Vagrantfile
-	vagrant destroy
-	sed -ie '/# VAGRANT START/,/# VAGRANT END/d' ~/.ssh/config
-	sed -ie '/IML Vagrant cluster/d' ~/.ssh/authorized_keys
-	export LIBVIRT_DEFAULT_URI=qemu:///system;                       \
-	for net in intel-manager-for-lustre{0,1,2,3} vagrant-libvirt; do \
-	    virsh net-destroy $$net || true;                             \
-	    virsh net-undefine $$net || true;                            \
-	done
+	set -e;                                                              \
+	if [ -e $(VAGRANT_VM3_LIBVIRT_DIR)/id ]; then                        \
+	    echo "LIBVIRT detected as provider";                             \
+	    vagrant destroy;                                                 \
+	    sed -ie '/# VAGRANT START/,/# VAGRANT END/d' ~/.ssh/config;      \
+	    sed -ie '/IML Vagrant cluster/d' ~/.ssh/authorized_keys;         \
+	    export LIBVIRT_DEFAULT_URI=qemu:///system;                       \
+	    for net in intel-manager-for-lustre{0,1,2,3} vagrant-libvirt; do \
+	        virsh net-destroy $$net || true;                             \
+	        virsh net-undefine $$net || true;                            \
+	    done                                                             \
+	else                                                                 \
+	    echo "LIBVIRT not detected as provider";                         \
+	    vagrant destroy;                                                 \
+	    sed -ie '/# VAGRANT START/,/# VAGRANT END/d' ~/.ssh/config;      \
+	    sed -ie '/IML Vagrant cluster/d' ~/.ssh/authorized_keys;         \
```

Comment:
These 3 lines are duplicated from the _libvirt_ branch.  Why not move the branch until after these lines?